Automattic/wp-calypso
# GF Signup: Hide the new interval dropdown in untested contexts

Related to #

## Proposed Changes

* Some untested contexts result in bugs when shipping the dropdown change. This change hides it all known untested contexts and hides it by default if the parameter is not provided.

## Testing Instructions



* Go to a stepper flow `/setup/newsletter`
* Continue until the plans step
* Make sure the dropdown is not visible but the plan toggle is

## Pre-merge Checklist
